doc/visual-programming/source/loading-your-data/index.rst

```diff
@@ -2,77 +2,66 @@ Loading your Data
+-   Attribute names in the column header
     can be preceded with a label followed by a hash. Use c for class
     and m for meta attribute, i to ignore a column, and C, D, S for
     continuous, discrete and string attribute types. Examples: C\#mpg,
-    mS\#name, i\#dummy. Make sure to set **Import Options** in `File <http://docs.orange.biolab.si/widgets/rst/data/file.html#file>`_
-    widget and set the header to **Orange simplified header**.
--   Orange's native format is a tab-delimited text file with three
-    header rows. The first row contains attribute names, the second the
-    type (**continuous**, **discrete** or **string**), and the third
-    the optional element (**class**, **meta** or **string**).
+    mS\#name, i\#dummy.
+-   An alternative to the hash notation is Orange's native format with three
+    header rows: the first with attribute names, the second specifying
+    the type (**continuous**, **discrete** or **string**), and the third
+    proving information on the kind of attribute (**class**, **meta** or **string**).
 
-Data from Excel
----------------
+An Example: Data from Excel
+---------------------------
 
-Orange 3.0 can read .xlsx files from Excel. Here is an example data set (:download:`sample.xlsx <sample.xlsx>`) in Excel:
+Here is an example data set (download it from :download:`sample.xlsx <sample.xlsx>`) as entered in Excel:
 
 .. image:: spreadsheet1.png
     :width: 600 px
     :align: center
 
-In Orange, let us start with a simple workflow with File and Data Table widgets,
+The file contains a header row, eight data instances (rows) and seven data attributes (columns). Empty cells in the table denote missing data entries. Rows represent genes; their function (class) is provided in the first column and their name in the second. The remaining columns store measurements that characterize each gene. With this data, we could, say, develop a classifier that would predict gene function from its characteristic measurements.
+
+Let us start with a simple workflow that reads the data and displays it in a table:
 
 .. image:: file-data-table-workflow.png
     :align: center
 
-and then load the data from Excel by opening the File widget (double click on the icon and of the widget) and click on the file browser icon ("..."),
+To load the data, open the File widget (double click on the icon of the widget), click on the file browser icon ("...") and locate the downloaded file (from :download:`sample.xlsx <sample.xlsx>`) on your disk:
 
 .. image:: loadingyourdata.png
-    :width: 600 px
+    :width: 401 px
     :align: center
 
-locate the data file (e.g. :download:`sample.xlsx <sample.xlsx>`) and open
-it. The **File** widget sends data to **Data Table** widget, which displays the following result:
+The **File** widget sends the data to **Data Table** widget, which displays it in the spreadsheet. Double click the **Data File** widget to see its contents:
```

Comment:
- to _the_ **Data Table** widget
- if it was not mentioned before it should be "a spreadsheet"
- I also don't know if "spreadsheet" is the best word for the "table" in the widget?
(maybe this second part of the first sentence can be omitted)
- **Data File** widget -> **Data Table** widget?